Assignment2/EnduEo-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,70 @@
+
+{	
+	"@context":{
+		"cl": "http://ontogrid.net/Ex1b/classes/",
+		"sn": "http://ontogrid.net/Ex1b/classes/sensors/",
+		"cp": "http://ontogrid.net/Ex1b/classes/computers/",
+		"ms": "http://ontogrid.net/Ex1b/classes/measurements/",
+		"us": "http://ontogrid.net/Ex1b/classes/users/",
+		"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"includes": "rdfs:label",
+		"hasMeasurement": "rdfs:label",
+		"hasOwner": "rdfs:label",
+		"hasName": "rdfs:label",
+		"hasTemperature": "rdfs:label",
+		"atTime":"rdfs:label"

```

Comment:
Check my comments on attribute definitions that I did in your JSONtoJSON-LD file.